h/migrations/versions/7f3d80550fff_correct_timestamps_of_elife_annotations.py

```diff
@@ -0,0 +1,6641 @@
+"""
+Correct the timestamps of imported eLife annotations.
+
+Set the timestamps of eLife annotations that were imported using an API script
+to their correct timestamps according to the data file that eLife gave us.
+
+Revision ID: 7f3d80550fff
+Revises: 9bcc39244e82
+Create Date: 2017-11-29 15:06:59.207780
+"""
+
+from __future__ import unicode_literals
+
+from datetime import datetime
+import logging
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+from h.db import types
+
+
+revision = '7f3d80550fff'
+down_revision = '9bcc39244e82'
+
+
+log = logging.getLogger(__name__)
+
+
+Base = declarative_base()
+Session = sessionmaker()
+
+
+# The string format of the timestamps below.
+FORMAT = "%Y-%m-%dT%H:%M:%SZ"
+
+
+class Annotation(Base):
+    __tablename__ = 'annotation'
+    id = sa.Column(types.URLSafeUUID, primary_key=True)
+    created = sa.Column(sa.DateTime)
+    updated = sa.Column(sa.DateTime)
+
+
+def upgrade():
+    session = Session(bind=op.get_bind())
+
+    # Pre-parse the timestamps to reduce the amount of time spent in the DB
+    # transaction.
+    for timestamp in TIMESTAMPS:
+        timestamp["created"] = datetime.strptime(timestamp["created"], FORMAT)
+        timestamp["updated"] = datetime.strptime(timestamp["updated"], FORMAT)
+
+    # Now make the DB changes.
+    corrected = 0
+    for timestamp in TIMESTAMPS:
+        query = session.query(Annotation)
+        query = query.filter(Annotation.id == timestamp["id"])
```

Comment:
Is there a difference between these three lines and `session.query(Annotation).get(timestamp["id"])`? If so, could we simplify this? Or is it more useful to maintain consistency between this version and the `imported_id`-based version?

Reply:
Actually no, there isn't a difference, I just forgot about `get`

Reply:
I've pushed this change